packages/app-desktop/gui/MenuBar.tsx

```diff
@@ -421,8 +443,18 @@ function useMenu(props: Props) {
 
 			function _showAbout() {
 				const v = versionInfo(packageInfo);
+				const pluginService = PluginService.instance();
+				const pluginItems = useInstalledPlugins(pluginService.plugins);
+				function random(pluginItems: PluginItem[]): string {
+					let content = v.message.concat('\n');
+					for (const item of pluginItems) {
+						content = content.concat(`\n${item.manifest.name}: ${item.manifest.version}`);
+					}
+					return content;
+				}
+				const message = random(pluginItems);
```

Comment:
why is it called random? Just sort and append the plugin name and version and that's it.